### PR TITLE
fix fixable flaws in Web/API/Window/requestIdleCallback

### PR DESCRIPTION
**Pull request by peterbe**
The page was using code inside the `<pre>` tag which all gets rendered away anyway.

### CODE REVIEW

**Comment by peterbe** [comment]
Confusingly, none of those `macros` flaws are new. They come from the sidebar I think.

File: files/en-us/web/api/window/requestidlecallback/index.html [hunk 1]
```diff
@@ -32,7 +32,7 @@
 <h2 id="Syntax">Syntax</h2>
 
 <pre
-  class="brush: js"><code>var <em>handle</em> = <em>window</em>.requestIdleCallback(<em>callback</em>[, <em>options</em>])</code></pre>
+  class="brush: js">var handle = window.requestIdleCallback(callback[, options])</pre>
```
Comment (jpmedley) [inline files/en-us/web/api/window/requestidlecallback/index.html:35]:
```suggestion
  class="brush: js">window.requestIdleCallback(callback);<br/>
  window.requestIdleCallback(callback, options);</pre>
```

Comment (jpmedley) [reply files/en-us/web/api/window/requestidlecallback/index.html:35]: The [syntax rules](https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections) have recently changed.

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/window/requestidlecallback/index.html (hunk 1 above)
Comment (peterbe) [reply files/en-us/web/api/window/requestidlecallback/index.html]: Thanks! 
But no need for a `<br/>` tag inside `<pre>`. 

Updated: https://pr5165.content.dev.mdn.mozit.cloud/en-US/docs/Web/API/Window/requestIdleCALLback#syntax
(cache busting the URL see my latest update)

**Comment by sideshowbarker** [comment]
@jpmedley This PR seems to be waiting on your re-review/approval of changes made in response to your review comments

**Review by jpmedley — APPROVED** [review]

File: files/en-us/web/api/window/requestidlecallback/index.html [hunk 2]
```diff
@@ -2,14 +2,14 @@
...
-- requestIdleCallback
-- polyfill
+  - API
+  - HTML DOM
+  - JavaScript timers
+  - Method
+  - Reference
+  - Window
+  - requestIdleCallback
+  - polyfill
```
Comment (jpmedley) [inline files/en-us/web/api/window/requestidlecallback/index.html:12]:
```suggestion
  - polyfill
browser-compat: api.Window.requestIdleCallback
```

**Review by jpmedley — CHANGES_REQUESTED** [review]
Sorry for the delay. I've been out of the office. 

The suggestion I just made should resolve the merge conflict.

File: files/en-us/web/api/window/requestidlecallback/index.html (hunk 2 above)
Comment (peterbe) [reply files/en-us/web/api/window/requestidlecallback/index.html]: Resolved now.

**Comment by peterbe** [comment]
Back to you @jpmedley

**Review by jpmedley — APPROVED** [review]

**Pull request merged**